content/usgs-packages/sbtools_Assessment.Rmd

```diff
@@ -39,11 +39,72 @@ knit_hooks$set(addToggle = function(before, options, envir) {
 set.seed(1)
 ```
 
+```{r sbtools-auth, echo=FALSE}
+# run vizlab::storeSBcreds() once before this can work
+home <- path.expand('~')
+sbCreds <- file.path(home, ".vizlab/sbCreds")
+credList <- readRDS(sbCreds)
+un <- rawToChar(credList$username)
+pw <- rawToChar(credList$password)
+sbtools::authenticate_sb(un, pw)
+```
+
 Before starting the exercises, you should make sure that the `sbtools` package is installed and loaded. If you haven't recently updated, you could reinstall the package by running `install.packages('sbtools')` or go to the "Update" button in the "Packages" tab in RStudio.
 
 ```{r load-sbtools}
 # load the sbtools package
 library(sbtools)
 ```
 
+## Exercise 1
+
+Using querying functions, find out how many ScienceBase items exist related to lake temperature during 1992. Hint: use `query_sb` to search using more than one criteria.
```

Comment:
Specify which type of date is acceptable? Default is dateType='lastUpdate', I think.